### PR TITLE
Fix a 'bug' that new mods would be listed as updated

### LINKED ISSUE #73: Suggestion:  Updated != New
The front page has a "Newest Mods" and a "Recently Updated Mods" section.

A long-standing pet peeve of mine:  Currently (and on Kerbal Stuff, since forever), it treats a new mod as being "updated", which means any time someone posts a new mod, it appears twice on the page (at the top of both the "new" and "updated" lists).

Suggestion:  Alter the logic for "Recently Updated" so that it *only* shows mods that have more than one version.

Rationale:  There's not a lot of use in a mod showing up twice on the page.  It means that the existing mods who are actually *updated* don't get as much screen time because the new mods are elbowing them out.  Seems more helpful to treat "new" and "updated" as different things, since both sections are present.

Thoughts, anyone?

### PR DESCRIPTION
**Pull request by RockyTV**
Every recently published mod would have the 'updated' field set to the publish date, which would display it twice in the homepage: both in recently updated and newly created mods.

It's a hacky way to fix #73. The other method involves messing with SQL and figuring out how to count the versions for each recently updated mod, or have a new field in the Mod object that would count the versions it has.

### CODE REVIEW

**Comment by FrikFry** [comment]
It seems like there should be a way to make sqlalchemy count the number of mod.versions and filter the new/recent lists on that, but I'll be damned if I can figure it out. Anyone know a bunch about sqlAlchemy filter statements?

**Comment by RockyTV** [comment]
Updated the pull request with a less hacky method.

File: KerbalStuff/blueprints/anonymous.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ def index():
     featured = Featured.query.order_by(desc(Featured.created)).limit(6)[:6]
     top = search_mods("", 1, 3)[0]
     new = Mod.query.filter(Mod.published).order_by(desc(Mod.created)).limit(3)[:3]
```
Comment (FrikFry) [inline KerbalStuff/blueprints/anonymous.py]: Would it make sense to limit new mods to mods that are truly new, IE only have one version?

**Comment by RockyTV** [comment]
Perhaps a more sane patch would be to only display mods with one version in the new mods section/page, and display mods with more than a version on recent.

**Comment by FrikFry** [comment]
That occurred to me as well, but I'm not sure it would be worth the extra effort. Publishing or updating a mod doesn't seem to change the value of its created attribute. With that in mind, it seems like the only time this would come up would be if someone uploaded a mod and immediately published a new version, in which case, it's still a new mod and probably deserves to still be listed as such. I suppose someone could take advantage of that to get their mod listed twice, but I'm not sure how big of a concern that is.

I think what you've done so far is good enough. :+1:

**Pull request merged**